genon/preprocessor/facade/enrichment/table_description.py

```
@@ -0,0 +1,815 @@
+        safe_summary = doc_summary or "-"
+        tpl = self._refine_prompt_tpl if use_refine else self._summary_prompt_tpl
+        try:
+            prompt = tpl.render(
+                before_context=safe_before,
+                after_context=safe_after,
+                caption=safe_caption,
+                section_header=safe_header,
+                doc_summary=safe_summary,
+                **(self.options.variables or {}),
+            )
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)